k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -89,6 +89,10 @@ pub struct UpgradeArgs {
     #[clap(global = true, long, short = 'R')]
     pub skip_replica_rebuild: bool,
 
+    /// If set then upgrade will skip the cordoned node validation
```

Comment:
missing period `.`
also please fix the other ones above

Reply:
done